content/shmem_signal_wait_until.tex

```diff
@@ -11,7 +11,7 @@
 
 \begin{apiarguments}
 
-\apiargument{IN}{sig\_addr}{Local address of the source signal variable.}
+\apiargument{IN}{sig\_addr}{Local address of the remotely accessible source signal variable.}
```

Comment:
Is this a new change?

Reply:
This was there in the original PR commit here, if that is what you are asking. I thought this can be treated as a doc edit.